core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -2095,15 +1942,17 @@ private ExprValue getTimeDifference(String part, LocalDateTime startTime, LocalD
   private ExprValue exprTimestampDiff(
       ExprValue partExpr, ExprValue startTimeExpr, ExprValue endTimeExpr) {
     return getTimeDifference(
-        partExpr.stringValue(), startTimeExpr.datetimeValue(), endTimeExpr.datetimeValue());
+        partExpr.stringValue(),
+        LocalDateTime.ofInstant(startTimeExpr.timestampValue(), ZoneOffset.UTC),
```

Comment:
Can you get `ZoneOffset` from `UTC_ZONE_ID`? If yes, please, use it everywhere. If no, please add a new constant. We should be able to easy change it everywhere or make a variable.

Reply:
updated in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/f6edc6be62abbe3a3974665e8d51c515584234e0, although there are a few functions involving utilizing `ZoneOffset.UTC` to get `EpochSeconds` that have left as they are as I would have to go through and change more functions that I haven't touched yet